packages/augur-sdk/src/state/db/DisputeDB.ts

```diff
@@ -9,16 +9,16 @@ export class DisputeDatabase extends DerivedDB {
     protected processDoc(log: ParsedLog): ParsedLog {
         if (log.name === 'InitialReportSubmitted') {
             return this.processInitialReportSubmitted(log);
-        } else if (log.name === 'DisputeCrowdsourcerCompleted') {
-            return this.processDisputeCrowdsourcerCompleted(log);
+        } else if (log.name === 'DisputeCrowdsourcerCreated') {
+            return this.processDisputeCrowdsourcerCreated(log);
         } else if (log.name === 'InitialDisputeCrowdsourcerContribution') {
```

Comment:
This also needs to be changed to "DisputeCrowdsourcerContribution` and `processDisputeCrowdsourcerContribution`